index.js

```diff
@@ -57,6 +57,10 @@ Benchmarked.prototype.defaults = function(benchmarked) {
     }
   };
 
+  this.format = function(benchmark) {
+    return '  ' + benchmark;
+  };
```

Comment:
I just noticed this... I think it should be changed to the following to ensure the `format` function is always on `opts` when used later.
```js
if (typeof this.options.format !== 'function') {
  this.options.format = function(benchmark) {
    return '  ' + benchmark;
  };
}
```